### ISSUE TITLE
V2.0: incorrect logs on single_chan_pkt_fwd

### ISSUE THREAD

**Issue by cdupont** [Waziup/WaziGate#80]
The logs of single_chan_pkt_fwd seems incomplete:
![image](https://user-images.githubusercontent.com/484033/89180650-7ef2bc00-d592-11ea-94b9-2356723a0f04.png)

It seems to log only ERR. The complete log is:
```
$ docker logs -f d509e03115d9
[ERR  ] Can not set output power: expected 0xe, got 0xfc
[     ] waiting for packets ...
[     ] (<- 172.18.0.3:1700) PullAck: Token: 3D8E
[     ] listening on [::]:58990
[     ] (-> 172.18.0.6:1700) PullData: Token: BD8, Gateway ID: AA555A0000000000
[     ] radio SX1276 activated.
[ERR  ] Can not set output power: expected 0xe, got 0xfc
[     ] waiting for packets ...
[     ] (<- 172.18.0.6:1700) PullAck: Token: BD8
```

Also FATAL doesn't seems to be logged.

**Comment by mojtaba-esk** [comment]
The exact same command is used to generate logs, you do not see it because it shows only the latest 50 lines of logs in the log window

**Comment by cdupont** [comment]
Today I reopenned, but the logs window is empty:

![image](https://user-images.githubusercontent.com/484033/99662282-55886e00-2a65-11eb-8c56-ca0f9cdf34e6.png)

The request seems correct, but the "response" is empty.
![image](https://user-images.githubusercontent.com/484033/99662234-46092500-2a65-11eb-971b-37b236a98a9d.png)

The ID of the container is correct in the request.

**Comment by mojtaba-esk** [comment]
Maybe nothing is going on there or the container has not started

**Comment by cdupont** [comment]
Does it display only new logs?
From command line I can can some logs:
```
$ docker logs waziup.wazigate-lora -f
GET device/id: 200 OK
Gateway ID: 0242ac120003ccde
Proxy listening on 'proxy.sock'.
--- Init ChirpStack
Organization "wazigate_1237770961" OK.
Network-server "Local Chirpstack Network Server" OK.
Service-profile "Wazigate Service Profile" OK.
Gateway "LocalWazigate_0242ac120003ccde" OK.
Application "Wazigate" OK.
Device-profile "Wazidev" OK.
The ChirpStack data has not changed.
--- Init Device
GET devices?meta=lorawan: 200 OK
DevEUI AA555A002601143F -> Waziup ID 5fafd2b526840c0006a68bd0
Reactivating Chirpstack device ... OK
There are 1 LoRaWAN devices.
[@] 200, GET /dist/hook.js s:9670
[@] 404, GET /undefined s:19
[@] 200, GET /dist/hook.js s:9670
```

However, things are different on the UI:
![image](https://user-images.githubusercontent.com/484033/99676064-51b21700-2a78-11eb-8491-ce59481e2372.png)

**Comment by mojtaba-esk** [comment]
It uses something like this:
`docker logs --tail=50 waziup.wazigate-lora`

**Comment by cdupont** [comment]
@mojtaba-esk your command works... However the result on the UI is the same...

**Comment by cdupont** [comment]
Any update?

**Comment by cdupont** [comment]
I saw the new UI, great!
However, this specific bug is still there.

![image](https://user-images.githubusercontent.com/484033/100737355-578eed00-33d4-11eb-8271-110ad1b75026.png)

Only the line "The ChirpStack data has not changed." makes it through the WaziGate logs. Very strange.

**Comment by cdupont** [comment]
I think I understood, it seems that you display only "stdout". However most of the traces are made on stderr:
```
$ docker logs --tail=20 waziup.wazigate-lora -t   2> /dev/null
2020-11-30T17:31:52.968149322Z The ChirpStack data has not changed.
```
You could do something like this to get stderr as well:
```
docker logs --tail=20 waziup.wazigate-lora -t 2>&1
```

**Comment by mojtaba-esk** [comment]
Good point, let me try it

**Comment by cdupont** [comment]
Fixed in ISO 2.1.3, closing
